api/endpoints/zoom.js

```diff
@@ -15,10 +15,16 @@ export default async (req, res) => {
 
     // Let's lookup our webhook event to see if we already got this event.
 
-    let meeting;
-    if (req.body.payload.object.id) {
-      meeting = await Prisma.find('meeting', { where: { zoomID: req.body.payload.object.id.toString() }, include: { schedulingLink: true } })
+    const getZoomId = (req) => {
```

Comment:
Can this be pulled to outside this method for purposes of code cleanliness/clarity?  This method is fairly long already and would be easier to read without needing to worry about the details of how we extract this value safely.